includes/webhook.sh

```diff
@@ -33,4 +34,13 @@ send_start_notification() {
 }
 send_stop_notification() {
   send_webhook_notification "$WEBHOOK_STOP_TITLE" "$WEBHOOK_STOP_DESCRIPTION" "$WEBHOOK_STOP_COLOR"
-}
\ No newline at end of file
+}
+send_install_notification() {
```

Comment:
Do we really need ENVs for the user to change that?
I would be okay with it beeing hardcoded.